### PR TITLE
Cython 3

### PR DESCRIPTION
**Pull request by mattwthompson**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mattwthompson** [comment]
#121  is probably what we want, but both are failing because the tarball disappeared

File: recipe/meta.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ requirements:
   host:
     - python
     - setuptools
-    - cython
+    - cython =3
```
Comment (hainm) [inline recipe/meta.yaml:54]: I don't think we need `cython` for build ambertools.
You can try to remove it too see what happens.

Comment (mattwthompson) [reply recipe/meta.yaml:54]: Nice, #122 seems to indicate it works fine!

**Comment by mikemhenry** [comment]
We should just drop cython as a dep

**Pull request closed without merging**